tests/test_tokenizer_indexer.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import os
+import torch
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+# import pytest
+import unittest
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.common import cached_transformers
+from gector.tokenizer_indexer import PretrainedBertIndexer
+from allennlp.data.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+from allennlp.data import Vocabulary
+
+class TestPretrainedTransformerIndexer(unittest.TestCase):
+    def test_do_lowercase(self):
+        tokenizer = WhitespaceTokenizer()
+        sentence = "the Quick brown fox jumped over the laziest lazy elmo"
+        vocab_path = "../data/output_vocabulary"
+        tokens = tokenizer.tokenize(sentence)
+        vocab = Vocabulary.from_files(vocab_path)
+        model_name = 'roberta-base'
```

Comment:
You seem to be repeating these lines in almost every test? May be refactor this into a `setup()` method for the test class?